deku-derive/src/macros/deku_write.rs

```diff
@@ -98,10 +98,13 @@ fn emit_struct(input: &DekuData) -> Result<TokenStream, syn::Error> {
         }
     };
 
+    // avoid outputing `use core::convert::TryInto` if update() function is empty
+    let update_use = check_use_update(&field_updates);
```

Comment:
`check_update_use` to stay consistent with variable name?

Reply:
:+1:  done